coord/src/module.c

```diff
@@ -1528,6 +1528,14 @@ static int searchResultReducer(struct MRCtx *mc, int count, MRReply **replies) {
   return res;
 }
 
+static inline bool cannotBlockCtx(RedisModuleCtx *ctx) {
+  return RedisModule_GetContextFlags(ctx) & REDISMODULE_CTX_FLAGS_DENY_BLOCKING;
+}
+
+static inline int ReplyBlockDeny(RedisModuleCtx *ctx, const RedisModuleString *cmd) {
+  return RMUtil_ReplyWithErrorFmt(ctx, "Cannot perform `%s`: Cannot block", RedisModule_StringPtrLen(cmd, NULL));
```

Comment:
"cannot perform %s in transactions or in LUA script" would be more informative for users IMO

Reply:
"transactions/LUA script/RM_Call call that denies blocking"

Reply:
"cannot perform %s in client blocking operations (such as transactions or LUA scripts)"